core/settings.py

```
@@ -0,0 +1,689 @@
+        {
+            "SCHEMA": "applications.graphQL.schema.schema",
+            "MIDDLEWARE": [
+                "graphene_django.debug.DjangoDebugMiddleware",
+            ],
+        },
+        environ=False,
+    )
+
+    #!SECTION End - GraphQL Settings (Graphene-Django)
+
+    # SECTION - Email Settings (Django-Mailer)
```

Pylint (3.3.7)

[warning] SECTION - Email Settings (Django-Mailer)

(W0511)